doc/connectivity/bluetooth/api/mesh/rpr_cli.rst

```diff
@@ -0,0 +1,132 @@
+element.
+
+Scanning
+********
+
+The scanning procedure is used to scan for unprovisioned devices located nearby the Remote Provisioning Server. The Remote Provisioning Client starts a scan procedure by using the :c:func:`bt_mesh_rpr_scan_start` call:
+
+.. code-block:: C
+
+      static void rpr_scan_report(struct bt_mesh_rpr_cli *cli,
+                  const struct bt_mesh_rpr_node *srv,
+                  struct bt_mesh_rpr_unprov *unprov,
+                  struct net_buf_simple *adv_data)
+      {
+
+      }
+
+      struct bt_mesh_rpr_cli rpr_cli = {
+         .scan_report = rpr_scan_report,
+      };
+
+      const struct bt_mesh_rpr_node srv = {
+         .addr = 0x0004,
+         .net_idx = 0,
+         .ttl = BT_MESH_TTL_DEFAULT,
+      };
+
+      struct bt_mesh_rpr_scan_status status;
+      uint8_t *uuid = NULL;
+      uint8_t timeout = 10;
+      uint8_t max_devs = 3;
+
+      bt_mesh_rpr_scan_start(&rpr_cli, &srv, uuid, timeout, max_devs, &status);
+
+The above example shows pseudo code for starting a scan procedure on the target RPR server node. This
+procedure will start a ten second multiple devices scanning where the generated scan report will contain
+a maximum of three unprovisioned device. If the UUID argument was specified, the same procedure would
+only scan for the device with the corresponding UUID. After the procedure completes the
+server will send the scan report that will be handled in the clients :c:member:`bt_mesh_rpr_cli.scan_report` callback.
+
+Additionally, the Remote Provisioning Client model also supports extended scanning with the
+:c:func:`bt_mesh_rpr_scan_start_ext` call. Extended scanning supplements regular scanning by allowing the
+Remote Provisioning Server to report additional data for a specific device. The Remote Provisioning Server will use active scanning to request
+a scan response from the unprovisioned device if it is supported by the unprovisioned device.
+
+Provisioning
+************
+
+The Remote Provisioning Client starts a provisioning procedure by using the :c:func:`bt_mesh_provision_remote` call:
+
+.. code-block:: C
+
+      struct bt_mesh_rpr_cli rpr_cli;
+
+      const struct bt_mesh_rpr_node srv = {
+         .addr = 0x0004,
+         .net_idx = 0,
+         .ttl = BT_MESH_TTL_DEFAULT,
+      };
+
+      uint8_t uuid[16] = { 0xaa };
+      uint16_t addr = 0x0006;
+      uint16_t net_idx = 0;
+
+      bt_mesh_provision_remote(&rpr_cli, &srv, uuid, net_idx, addr);
+
+The above example shows pseudo code for remotely provisioning a device through a RPR server node. This
+procedure will attempt to provision the device with the corresponding UUID, and assign the address 0x0006
+to its primary element using the network key located at index zero.
+
+Please note that during remote provisioning the same :c:struct:`bt_mesh_prov` callbacks are triggered as for ordinary
+provisioning. Please refer to section :ref:`bluetooth_mesh_provisioning` for further details.
+
+Re-Provisioning
```

Comment:
I am not sure if Zephyr doc uses title case for headings, there are too many inconsistencies to actually tell. So we stick to what we know from NCS (sentence case in headings).
```suggestion
Re-provisioning
```